pandas/core/groupby.py

```diff
@@ -1888,6 +1897,15 @@ def group_info(self):
         comp_ids = _ensure_int64(comp_ids)
         return comp_ids, obs_group_ids, ngroups
 
+    # 17530
```

Comment:
put a comment / doc-string  inside the function on what this is doing(don't need to add the issue number)